staging/src/k8s.io/client-go/rest/request_test.go

```diff
@@ -306,7 +306,7 @@ func TestRequestBody(t *testing.T) {
 	if err != nil {
 		t.Fatalf("unable to create temp file")
 	}
-	defer f.Close()
+	f.Close()
```

Comment:
do you know why we need a temp file here is it is never used afterwards?

Reply:
We just construct a tmp file to get a file name as string type to test Body() func's different case, and remove it before call Body() to get an expected err.